run_action.py

```diff
@@ -174,28 +174,63 @@ def main():
 
     github_token = os.environ.get("INPUT_GITHUB_TOKEN")
     github_api_url = os.environ.get("GITHUB_API_URL")
+    
     request_url = "%s/repos/%s/issues/%s/comments" % (
         github_api_url,
         repo,
         args.pull_request_id,
     )
 
+    headers = {
+        "Authorization": "token %s" % github_token,
+    }
+    report = {
+        "body": message
+    }
+
+    one_comment = os.environ.get("INPUT_ONE_COMMENT")
+    is_update = False
+    if one_comment.lower() in ("true", "1"):
+        ## Search comments
+        pr_comments = requests.get(request_url,
+                                   headers=headers
+                                   ).json()
+                                
+        header_message = "## 📌 Duplicate code detection tool report\n"
+        for pr_comment in pr_comments[::-1]:
+
+            if header_message in pr_comment["body"]: ## Search the bot's comment
```

Comment:
Let's be specific on where the header should be contained

```suggestion
            if pr_comment["body"].startswith(header_message) : ## Search the bot's comment
```